### DISCUSSION TITLE
Manim Presentation

### DISCUSSION THREAD

**Discussion by hackyfemmyboi** [ManimCommunity/manim#2317] [Q&A]
I tried getting Manim-Presentation, but it wouldnt show up in the plugins folder

`C:\Users\Rene>python -m manim plugins -l
Manim Community v0.12.0

Plugins:
 • manim_physics

C:\Users\Rene>`

**Comment by citrusmunch** [comment]
I tried to help someone with an issue like this.
`pip list` showed that `manim-presentation` was installed, but for whatever reason the binary was unavailable.
Probably worth filing an issue at the `manim-presentation` [repo](https://github.com/galatolofederico/manim-presentation).
